backend/hct_mis_api/apps/household/migrations/0172_migration.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.24 on 2024-03-13 15:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('household', '0171_migration'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='household',
+            name='social_worker',
```

Comment:
do not use `social_worker`, let add a type field with value "STANDARD" and "SINGLE".
Default value is "STANDARD" use "SINGLE" for any "fake" HH created.

Reply:
changed